### DISCUSSION TITLE
Working in RTSS osd support

### DISCUSSION THREAD

**Discussion by javguit** [LibreHardwareMonitor/LibreHardwareMonitor#538] [Ideas]
Hello, I had a working version of LHM and RTSS but lost it some time ago and didn't made a backup anywhere. I kind of remember how to remake it, but will it be accepted if I get it done? Just wondering because it kind of demands some work to get everything working as intended, but I'll be more than happy to do it again :)
The library I used to communicate with RTSS is https://github.com/spencerhakim/RTSSSharedMemoryNET
@PhyxionNL

**Comment by javguit** [comment]
Found a working version of my previous build, and was able to decompile it and add it to current build.
Still have to fix some stuff, but heres a pic of it working
![rtssTest](https://user-images.githubusercontent.com/63629902/133021249-07ab1117-2f87-4d59-9918-71cc95428cbd.jpg)

**Comment by javguit** [comment]
Is there a place to put a DLL or should I somehow link the repository where the DLL is at so I can create a pull request?

**Comment by AndyK70** [comment]
Hi there javguit,

would it be possible for you to develop a way for the RTSS OverlayEditor plugin to use the LHM sensor data as data source instead of injecting it in the shared memory?
The shared memory approach is the old variant and most of the new overlays are made with the OverlayEditor plugin of RTSS. It's way easier, more flexible and has a ton of features that the old method by shared memory injection does not provide.
You should find everything needed in the folder
"C:\Program Files (x86)\RivaTuner Statistics Server\SDK\Plugins\Client\OverlayEditor"

If it's not possible for you, that's fine, it would be nice to have.

**Comment by ozturkkl** [comment]
The OverlayEditor also uses values from the shared memory. Is there an update here? Did anyone find a solution or a workaround? I stopped using HWINFO since they introduced the 12 hour limit :(
